source/reactive-streams.txt

```diff
@@ -29,6 +29,10 @@ or set up a runnable project by following our tutorials.
 
 - `Source Code <https://github.com/mongodb/mongo-java-driver/tree/master/driver-reactive-streams/>`__
 
+If your Java application uses synchronous processing, use the
+:driver:`MongoDB Java Driver </java/sync/current/>` which makes synchronous
```

Comment:
Issue:
1) I think "uses synchronous processing" might be ambiguous in meaning.
2) It might not be entirely accurate because apps that use reactive streams can also use synchronous/blocking calls, especially if they do more than communicate with MongoDB through the Reactive Streams driver.

Suggestion:
I think it could be helpful to mention the synchronous calls are made between the app and MongoDB in the sync driver.

E.g. 
If you need to make synchronous calls between your application and MongoDB, use the MongoDB Java Driver instead.